packages/editor/web/src/RichContentEditor/RichContentEditor.tsx

```diff
@@ -400,9 +401,16 @@ class RichContentEditor extends Component<RichContentEditorProps, State> {
       innerModal: { openInnerModal: this.openInnerModal, closeInnerModal: this.closeInnerModal },
       renderInnerRCE: this.renderInnerRCE,
       innerRCERenderedIn,
+      disableKeyboardEvents: this.disableKeyboardEvents,
     };
   };
 
+  disableKeyboardEvents = shouldDisable => {
+    if (!this.props.isInnerRCE) {
+      this.readOnly = shouldDisable;
+    }
```

Comment:
This is used in render. So should probably be in state
```suggestion
    if (!this.props.isInnerRCE && readOnly !== this.state.readOnly) {
      this.setState({disableKeyboardEvents: shouldDisable});
    }
```